### ISSUE TITLE
Incorrect toolbar and tooltip positioning when editing element inside iframe

### ISSUE THREAD

**Issue by UCIS** [froala/wysiwyg-editor#1619]
We would like to use the Froala editor to edit individual elements inside an iframe, while keeping the Froala scripts and CSS outside of the iframe.

##### Expected behavior.
The editor toolbar should follow the iframe scroll position. Preferably the tooltip should be positioned outside of the iframe so it if not affected by inner styling and bounds.

##### Actual behavior.
The froala toolbar does not take the iframe's scroll position into consideration and the tooltip is often clipped by the iframe.

##### Steps to reproduce the problem.
The jsfiddle shows the problem, as well as some (commented) workarounds which we have tried.
https://jsfiddle.net/rdg5yL0p/3/

##### OS.
Linux Mint x64

##### Browser.
Google Chrome 54.0.2840.100 (64-bit)

**Comment by stefanneculai** [comment]
That is happening because you are setting the scrollable container as the body of the main page, while you're doing scroll inside the iframe.

**Comment by UCIS** [comment]
@stefanneculai I should have commented that one out in the example, but exactly the same problem occurs when the scrollable container is not set, as is shown here: https://jsfiddle.net/rdg5yL0p/7/

1. Click the text in the iframe, toolbar appears, scroll iframe, toolbar stays in the same position.
2. Scroll the iframe down, click the text in the iframe, toolbar appears in the wrong position.
3. Hover over the toolbar buttons. Tooltip appears inside the iframe, in the correct position (if the toolbar would be positioned correctly). If the cursor is near the (lower) edge of the iframe, the tooltip is clipped by the iframe bounds, even though the toolbar is visible.

**Comment by UCIS** [comment]
As we have a commercial license I was able to fix the problem in the original source. Can I post the patch here? The changes we had to make:

- Create the tooltip inside the current window.document, not inside the target document.
- In the at function, if the o_win.document != $container[0]ownerDocument, subtract the o_win scrollTop and scrollLeft (2*).
- Bind the showInline function to the scroll event on o_win.
